charts/karpenter/templates/deployment.yaml

```diff
@@ -98,6 +98,50 @@ spec:
                   containerName: controller
                   divisor: "0"
                   resource: limits.memory
+          {{- with .Values.settings.batchMaxDuration }}
```

Comment:
I think `FEATURE_GATES` needs to be added here, but otherwise this looks good to me.